docker/modify-gtf/SetGeneNameToId.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+
+import argparse
+import re
+
+parser = argparse.ArgumentParser(description="Set the gene_name within a gtf to be equivalent to the values within gene_id.")
+parser.add_argument('--in-gtf-file', dest='ingtf', help='input gtf file')
+parser.add_argument('--out-gtf-file', dest='outgtf', help='output gtf file')
+
+args = parser.parse_args()
+
+
+def setGeneNameToId(in_gtf, out_gtf, verbose=True):
+    with open(in_gtf, 'r') as fpin, open(out_gtf, 'w') as fout:
```

Comment:
`fpin` to me means `file pointer in`, so shouldn't `fout` be `fpout`?

Reply:
Changed!